src/templates/zh-CN/2019/base.html

```diff
@@ -0,0 +1,203 @@
+    "resource-hints": "资源提示",
+    "http2": "HTTP/2"
+  }
+%}
+
+{% set localizedTeamNames = {
+  "analysts": "分析者",
+  "authors": "作者",
+  "brainstormers": "头脑风暴者",
+  "designers": "设计者",
+  "developers": "开发者",
+  "editors": "编辑",
+  "leads": "领导者",
+  "reviewers": "审稿者",
+  "translators": "翻译者", 
+  }
+%}
+
+{% set localizedContributors = {
+  "kleinab": 'Abigail Klein 是谷歌的软件工程师. 她致力于谷歌文档、表格和幻灯片的web可访问性 她为谷歌幻灯片添加了<a href="https://www.blog.google/outreach-initiatives/accessibility/whats-you-say-present-captions-google-slides/">自动字幕</a>,并改进了屏幕阅读器、盲文、屏幕放大镜和高对比度支持。 她目前致力于谷歌Chrome和ChromeOS的可访问性。她拥有麻省理工学院(MIT)计算机科学学士和硕士学位，在那里她参与创立了一个辅助技术黑客马拉松，她还是该辅助技术课程的实验室助理和客座讲师。',
+  "argyleink": 'Adam Argyle 是一个谷歌Chrome开发者关系成员,聚焦于CSS; 他是一个对伟大的用户体验和UI贪得无厌的网络成瘾者; 在网上找到他 <a href="https://twitter.com/argyleink">@argyleink</a> 或者去他的网站 <a href="https://nerdy.dev">https://nerdy.dev</a>;',
+  "alankent": 'Alan Kent 是是谷歌的开发人员倡导者，致力于电商和内容生态，他的博客是 <a href="https://alankent.me">alankent.me</a> 推特是 <a href="https://twitter.com/akent99">@akent99</a>.',
+  "amedina": 'Alberto Medina 是谷歌Web内容生态系统小组的Developer Advocate，致力于通过AMP等先进技术和现代Web API的使用来促进网络上高质量内容的传播。Alberto目前的工作全力集中于内容管理系统，他领导的内容生态系统领域叫做CMS开发者关系',
+  "andydavies": 'Andy Davies 是一名自由职业网络性能顾问，曾帮助英国一些领先的零售商，报纸和金融服务公司提高网站速度。 他写了The Pocket Guide to Web Performance, 也是 Using WebPageTest的合著者，他也是伦敦Web性能聚会的组织者。你可以在推特找到 Andy <a href="https://twitter.com/andydavies">@AndyDavies</a>, 他也偶尔发布博客 <a href="https://andydavies.me">https://andydavies.me</a>',
+  "arturjanc": 'Artur Janc 是谷歌的信息安全工程师，致力于在谷歌和整个网络上设计和采用Web平台安全机制。他与网友在这里争辩 <a href="https://twitter.com/arturjanc">@arturjanc on Twitter</a>.',
+  "bazzadp": 'Barry Pollard 是一位软件开发者同时也是Manning book <a href="https://www.manning.com/books/http2-in-action">HTTP/2 in Action</a>的作者。他认为网络是神奇的，但希望让它变得更好。 你可以找到他的推特 <a href="https://twitter.com/tunetheweb">@tunetheweb</a> 和博客 <a href="https://www.tunetheweb.com">www.tunetheweb.com</a>.',
+  "bkardell": 'Brian Kardell是一名<a href="https://igalia.com">Igalia</a>的开发人员倡导者，标准贡献者， <a href="https://bkardell.com">博主</a>, 也同时是 W3C <a href="https://openjsf.org/">Open JS Foundation</a>咨询委员会代表. 他是可扩展网络社区小组的创始人和<a href="https://extensiblewebmanifesto.org">The Extensible Web Manifesto</a>的合著者',
+  "colinbendell": 'Colin 是 <a href="https://cloudinary.com/">Cloudinary</a>的CTO办公室的一部分， 并且是 O\'Reilly 书籍 <a href="https://www.oreilly.com/library/view/high-performance-images/9781491925799/">High Performance Images</a>的合著者。  他将大量时间花在大量数据，媒体，浏览器和标准的交汇处。 你可以关注他的推特 <a href="https://twitter.com/colinbendell">@colinbendell</a> 还有他的博客 <a href="https://bendell.ca/">https://bendell.ca</a>.',
+  "obto": 'David Fox 是LookZook的首席可用性研究人员和创始人，LookZook沉迷于发现有关构建满足用户期望的Web体验的所有一切。 他是一位网站心理学家，他深入研究网站不仅要学习用户正在苦苦挣扎的东西，而且要学习为什么以及如何最好地改善他们的体验。 他还是Google Chromium的撰稿人，演讲者以及网络研讨会和UX培训的提供者。',
+  "dougsillars": 'Doug Sillars 是从事性能和媒体交汇的自由数字游牧者。
+ 他的推特<a href="https://twitter.com/dougsillars">@dougsillars</a>, 同时时常在 <a href="https://dougsillars.com">dougsillars.com</a>发表博客',
```

Comment:
Remove extra new line:
```suggestion
  "dougsillars": 'Doug Sillars 是从事性能和媒体交汇的自由数字游牧者。他的推特<a href="https://twitter.com/dougsillars">@dougsillars</a>, 同时时常在 <a href="https://dougsillars.com">dougsillars.com</a>发表博客',
```